### DISCUSSION TITLE
Never got my .25 reward for finding block

### DISCUSSION THREAD

**Discussion by EVILTIM58** [Chia-Network/chia-blockchain#8218] [Support]
I found a block..   i was supposed to get .25 chia "automatically from the blockchain, but I did not.

![image](https://user-images.githubusercontent.com/89432385/130974892-000904bc-ea90-4ffc-af75-c62734c18c86.png)

![image](https://user-images.githubusercontent.com/89432385/130974956-5b9c9b83-a6b5-4111-a6bb-4c16a39695d1.png)

![image](https://user-images.githubusercontent.com/89432385/130975087-c3214cfd-08d1-42b4-87d0-cfe15496a3ee.png)

Ideas?

**Comment by emlowe** [comment]
Use a block explorer to check the block details. For block 638465 (https://www.chiaexplorer.com/blockchain/block/0x2443d920fd1f54fa57fc0ba11a8a091e96a3709a53c756a214b9a3c3954c3fd3)

The farmer reward address is `xch1v2k63wk2ujzm6fu6apl3pwlv9p3v3alg2stczd2pnp3dclvksauqw63ug5`

and the plot public key is `0xb7b97750bcda92c79921345fc16b3df1360bd32c9f0a246ccb1f908b1a38dc2d4930859e93914b38c8c8d618b0aba12f`

You can check if that is one of your plots 
You can also check your logs to see if your logs indicate you farmed a block.

**Comment by EVILTIM58** [comment]
it is, they do...

**Comment by EVILTIM58** [comment]
![image](https://user-images.githubusercontent.com/89432385/131007331-440890e3-8267-433b-8c25-80d7c6c5d213.png)

![image](https://user-images.githubusercontent.com/89432385/131007512-23987fad-6b0c-470d-8432-018107df2972.png)

**Reply by EVILTIM58** [reply]
my normal pool rewards are paying out fine..  just the .25 for the block is missing.

**Reply by loppefaaret** [reply]
in your last screenshot, it clearly states that the address used as farmer reward target address, belongs to another wallet - have you ever removed any keys, or manually edited these entries ?

**Comment by EVILTIM58** [comment]
I’m getting Pool rewards all day..

How do I fix it?

**Reply by loppefaaret** [reply]
yes, the pool rewards from plotnft pointing at a pool, is not affected by the "pool reward target address" - only OG plots solo farming are!

**Comment by EVILTIM58** [comment]
So how do I fix it

**Comment by loppefaaret** [comment]
`xch1v2k63wk2ujzm6fu6apl3pwlv9p3v3alg2stczd2pnp3dclvksauqw63ug5` is an address that is belonging to a key you do not have imported on the client - either you need to import the key that controls the wallet that address is from, or you need to update the "farmer/pool reward target address" to an address from a wallet that you got the key for

**Comment by EVILTIM58** [comment]
And where do I get that address from?  Everything I have points to this
address

**Reply by loppefaaret** [reply]
from CLI, doing a `chia keys show` will list the "First wallet address" of your key - 9/10 times, this would be used as a farmer/pool reward target address - but you can use _any_ wallet receive address

**Reply by AleksandrPDA** [reply]
Good evening! Did you manage to solve your problem?

**Comment by EVILTIM58** [comment]
is that first reward lost?

**Reply by loppefaaret** [reply]
if it is paid to a wallet, belonging to a key that you do not have, yes!

**Comment by plot14** [comment]
I have the same problem, I think we have all been defrauded. and there is no pre and post execution assistance for the chia project.

possible we can not go back to the various addresses and understand where the 0.25 xch end up?

**Comment by BrandtH22** [comment]
@EVILTIM58 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
